### PR TITLE
feat(EU): add setting charge limits

### PR DESCRIPTION
**Pull request by jwefers**
tested with EU Ioniq Electric

### CODE REVIEW

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 1]
```diff
@@ -462,6 +462,34 @@ def stop_charge(self, token: Token, vehicle: Vehicle) -> None:
...
+                    "targetSOClevel": dc_limit,
+                },
+                {
+                    "plugType": 1,
+                    "targetSOClevel": ac_limit,
+                },
+            ]
+        }
+        response = requests.post(url, json=body, headers=headers)
+        return str(response.status_code == 200)
```
[the remark this answers is not in the document]
Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:491]: @jwefers  just wanted to check on this.

**Review by cdnninja — CHANGES_REQUESTED** [review]

**Comment by cdnninja** [comment]
Are you able to resolve the merge conflicts and then we can merge?

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwefers) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:570]: @cdnninja i think there was no such header, but i'll confirm

[the remark this answers is not in the document]
Comment (jwefers) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:570]: unfortunately, EU api only returns an Etag and rate-limit state

**Comment by jwefers** [comment]
failing test is interesting, Fuats car response seems to miss `vehicleStatus.evStatus.reservChargeInfos.targetSOClist` in this instance

**Comment by CreasolTech** [comment]
@jwefers jwefers, how do you call set_charge_limits() ?
I'm very basic with python, and tried to use vm.set_charge_limits(vehicleId, evchargelimits) but I've found that your implementation in KiaUvoApiEU.py uses 
```set_charge_limits(self, token: Token, vehicle: Vehicle, ac_limit: int, dc_limit: int)```
instead of
```set_charge_limits(self, token: Token, vehicle: Vehicle, limits: EvChargeLimits)```  

Thanks a lot.

**Comment by jwefers** [comment]
@CreasolTech i refactored the method in an earlier PR which has since been merged, check https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/master/hyundai_kia_connect_api/Vehicle.py now, you can see the `EvChargeLimits`class there

**Comment by jwefers** [comment]
@fuatakgun So the test suite based on your car does not work for this EU feature - are you okay with merging this as is? Otherwise we could dig into adding Github secrets for my car for a EU test suite - some time in the future.

**Comment by CreasolTech** [comment]
> 

Are you sure? 
I can't find the set_charge_limit method on the current repository for Kia Europe.
https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/master/hyundai_kia_connect_api/KiaUvoApiEU.py

**Pull request merged**